src/mumble/main.cpp

```diff
@@ -380,6 +402,39 @@ int main(int argc, char **argv) {
 		}
 	}
 
+	// Populate extra translation dirs (directorys in which Mumble shall look for
+	// translation files that can overwrite the bundled one if present). The order
+	// in which they are added to this list determines the priority (first match
+	// will win).
+	// Start with the directory the Mumble executable lives in.
+	extraTranslationDirs.append(a.applicationDirPath());
+	// Next add directories in the config locations
+	for (const QString &currentConfigPath : QStandardPaths::standardLocations(QStandardPaths::ConfigLocation)) {
+		extraTranslationDirs.append(QDir::cleanPath(currentConfigPath + QDir::separator() + "MumbleTranslations"));
+	}
+	// Finally add a dir in the home directory
+	for (const QString &currentHomePath : QStandardPaths::standardLocations(QStandardPaths::HomeLocation)) {
+		extraTranslationDirs.append(QDir::cleanPath(currentHomePath + QDir::separator() + "MumbleTranslations"));
+	}
```

Comment:
I would use `QStandardPaths::AppDataLocation` instead.

Perhaps we could also omit `Mumble` in `MumbleTranslations`.

Reply:
I chose the home directory as it is very easily accessible on all platforms.

Adding `AppDataLocation` to the list sounds like a good idea though :+1: 

Should probably go before the config options (aka: priority 2)

> Perhaps we could also omit Mumble in MumbleTranslations.

Only for the AppDataLocation as this is guaranteed to already contain "Mumble" in the directory structure. In that case though I think I would opt for not having a subdirectory at all.
In all other cases I think we should definitely keep the `Mumble` prefix in order to make it clear what this directory belongs to.